.github/workflows/update-web.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will automatically create a PR in deephaven-core to update the WEB_VERSION
+name: Update web version
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [update_web]
+jobs:
+  update-web:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout latest deephaven-core
+        uses: actions/checkout@v3
```

Comment:
The "Checkout latest deephaven-core" isn't technically what's going on.

https://github.com/actions/checkout

> When checking out the repository that triggered a workflow, this defaults to the reference or SHA for that event. Otherwise, uses the default branch.

If we always want to run this against main, we should set `ref: main`

I think this impl is _ok_ as-is, but then I would just choose "Checkout" or something simpler for the name of this step.